log4j-core/src/test/java/org/apache/logging/log4j/core/net/SmtpManagerTest.java

```diff
@@ -39,14 +39,15 @@
 public class SmtpManagerTest {
 
     @Test
-    void testCreateManagerName() {
-        String managerName = SmtpManager.createManagerName("to", "cc", null, "from", null, "LOG4J2-3107",
-                "proto", "smtp.log4j.com", 4711, "username", false, "filter");
+    public void testCreateManagerName() {
```

Comment:
These tests were disabled (non-public).

Reply:
JUnit Jupiter doesn't need tests to be public, I think it's actually recommended to use the default visibility